public/docs/md/workflow/en/experiment-learnings.md

```diff
@@ -113,6 +113,10 @@ If desired, adjust the experiment’s **confidence level**. The default is 95%.
 Lowering your experiment’s confidence level makes it more likely that your experiment achieves statistical significance, but the trade-off is that doing so increases the likelihood of a false positive.
 {{/partial:admonition}}
 
```

Comment:
Should I add anything to this doc https://amplitude.com/docs/analytics/charts/experiment-results/experiment-results-dig-deeper?

Reply:
If anything, maybe just a link to other group by content